### PR TITLE
Adding r-bactdating

### PR DESCRIPTION
**Pull request by dnanto**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by dnanto** [comment]
Seems to be all good except for this [win_64 issue](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=204544&view=logs&j=240f1fee-52bc-5498-a14a-8361bde76ba0&t=23010c01-6a9d-5f94-9037-792618644b77&l=721). I currently have the build artifacts hosted on my [anaconda channel](https://anaconda.org/dnanto/r-bactdating) and it works with my Snakemake workflow.

File: recipes/r-bactdating/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+  version: {{ version|replace("-", "_") }}
+
+source:
+
+  url: https://github.com/xavierdidelot/BactDating/archive/v1.0.tar.gz
+  sha256: add9c391709e4d340d173cfb29d30ec6815d5ce89ff8038cea981260be01b151
+
+build:
+  merge_build_host: True  # [win]
+  # If this is a new build for the same version, increment the build number.
```
Comment (dbast) [inline recipes/r-bactdating/meta.yaml:17]:
```suggestion
```
Please remove all comments except the cran meta data at the end of the meta.yaml.

File: recipes/r-bactdating/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+# Suggests: knitr, testthat, covr
+requirements:
+  build:
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ compiler('cxx') }}            # [not win]
+    - {{ compiler('m2w64_cxx') }}      # [win]
+    - {{ posix }}filesystem        # [win]
+    - {{ posix }}git
```
Comment (dbast) [inline recipes/r-bactdating/meta.yaml:34]: Why is git required here? Is this a leftover?

Consider re-creating the recipe via https://github.com/bgruening/conda_r_skeleton_helper 

That remove not required comments and gets the requirements right.

Comment (dnanto) [reply recipes/r-bactdating/meta.yaml:34]: Ah, thanks, yeah, it's a leftover, I ran `conda skeleton` manually, I'll retry with the helper.

Comment (dnanto) [reply recipes/r-bactdating/meta.yaml]: This package isn't on the CRAN, so I'm applying suggestions/changes manually...

Comment (dnanto) [reply recipes/r-bactdating/meta.yaml]: all checks passed for the latest

**Comment by dbast** [comment]
@dnanto Thanks. Welcome at conda-forge!

**Pull request merged**